.gitignore

```diff
@@ -6,6 +6,8 @@ build
 .idea
 __pycache__
 tests/test_django/db.sqlite3
+htmlcov
+.coverage
 
 # pytest-cov
 .coverage*
```

Comment:
There is no need to add `.coverage` when '.coverage*' already include `.coverage`